### DISCUSSION TITLE
GitHub Bonuses dont work

### DISCUSSION THREAD

**Discussion by DefinetlyNotAI** [community/community#138026] [GitHub Education]
### Hi everyone,

Hi, i have been verified and can use copilot etc

But many other features just dont work:

## Atsra
```
You are not eligible for Astra Security's Student pack

An Error Was Encountered
```
Clicking go back: 
```
Authorize Astra Security - Student Developer Pack
@DefinetlyNotAIAstra Security - Student Developer Pack by [Astra Security](https://github.com/getastra)
wants to access your DefinetlyNotAI account
Personal user data
Full access
Authorizing will redirect to
https://dash.getastra.com

Not owned or operated by GitHub
Created 5 years ago
Fewer than 1K GitHub users
[Learn more about OAuth](https://docs.github.com/articles/authorizing-oauth-apps)
```

## Blackfire

```
Builds not available
Your current subscription does not provide builds. Please contact us if you are interested in our builds feature.
```

Keeps giving these messages for every feature! Cant use anything...
```
Your current subscription does not provide Continuous Profiling. Please contact us if you are interested in our Continuous Profiling feature.


Your current subscription does not provide Monitoring. Please contact us if you are interested in our Monitoring feature.
```

## Relic

When sigining in using github auth
```
No user found. Contact your administrator to grant you access, or you can [sign up](https://newrelic.com/signup?via=login) for a new organization.
```

For these im using my main email (non-school) that is from my account, should i switch to the school email for all these??

**Comment by alx-mp4** [comment]
Hi @DefinetlyNotAI, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!

**Comment by DefinetlyNotAI** [comment]
I am verfieied tho
